Makefile

```diff
@@ -169,6 +169,8 @@ deps: check-python-version build-cairo-2-compiler build-cairo-1-compiler
 	cargo install cargo-nextest --version 0.9.49
 
 deps-macos: check-python-version build-cairo-2-compiler-macos build-cairo-1-compiler-macos
+	brew install python@3.9
+	brew install pyenv
```

Comment:
Shouldn't `pyenv` be enough?

Reply:
No, it doesn't. The first time I saw the error, I thought the same thing, so I just installed the pyenv package, but the python3.9 error persists. 
![image](https://github.com/lambdaclass/starknet_in_rust/assets/146965181/fee9e2e7-52fa-4b85-b9f7-a9457567cd81)

Reply:
In my opinion, i don't like having this brew install commands in the make deps target.
won't be better to add this brew install commands to the README.md?
@Oppen @juanbono

Reply:
> No, it doesn't. The first time I saw the error, I thought the same thing, so I just installed the pyenv package, but the python3.9 error persists.

Then it's just not using the `pyenv` version. The `Makefile` should use `python` and `PATH` should be set to start with `$(pyenv root)/shims` so it find the right version based on `PYENV_VERSION`.
Example:
```Makefile
export PATH:="$(shell pyenv root)/shims:$(PATH)"
test:
    python -c 'print("hello")'
```

Reply:
I have tested it and it seems that the `pyenv is not necessary but when I run the tests I get the message `make[1]: pyenv: Command not found`, but they worked fine. I have removed the command to install `pyenv` from Makefile.

Reply:
Line 4 refers to `pyenv`. Either we assume `pyenv` will be used or we don't. If we don't, we need to specify what `python` needs to be.